### PR TITLE
Add specfile

### PR DESCRIPTION
**Pull request by fscheiner**
The RPM spec file is based on [Steve Traylen's one over at Fedora](https://src.fedoraproject.org/rpms/uberftp/blob/8046ad31980b197f8dc9879a3975d0db6717d882/f/uberftp.spec) and targets UberFTP version 2.9. It won't work until we create a tag named `Version_2_9`.

This PR includes my other PR #14.

### CODE REVIEW

**Comment by msalle** [comment]
Do we really need to have a spec file in the source tree? I normally prefer to keep software and packaging separate and let the former just create a dist tarball.

**Comment by fscheiner** [comment]
I understand your point, but where else should we put it? In the GCT for example we have a separate directory to host the RPM spec files and the metadata for Debian packaging.

**Comment by msalle** [comment]
We could make a packaging repository? That's quite common and keeps the code itself cleaner.

**Comment by matyasselmeci** [comment]
I suggest having the packaging directory be in the same repo as the software so we can keep the versions of the packages in sync with the version of the software. Doing a `make` should give you the same result as doing an `rpmbuild`.

**Comment by fscheiner** [comment]
@matyasselmeci 
> I suggest having the packaging directory be in the same repo as the software so we can keep the versions of the packages in sync with the version of the software. Doing a `make` should give you the same result as doing an `rpmbuild`.

So similar to what we do with the GCT.

@msalle
> We could make a packaging repository? That's quite common and keeps the code itself cleaner.

Actually I also wouldn't want to create a repository just for packaging, exactly for @matyasselmeci's reason.

**Comment by fscheiner** [comment]
@msalle, @matyasselmeci, @ellert, @maarten-litmaath
So how should we proceed from here?

**Comment by maarten-litmaath** [comment]
Hi all,
I think we just need to be practical here and propose
we go ahead with what Frank and Mat prefer, thanks!

**Comment by fscheiner** [comment]
> Hi all,
> I think we just need to be practical here and propose
> we go ahead with what Frank and Mat prefer, thanks!

@Mischa:
Ok for you?

**Comment by mischa** [comment]
Wrong Mischa

**Comment by fscheiner** [comment]
> Wrong Mischa

Ooops. Sorry.

> Hi all,
> I think we just need to be practical here and propose
> we go ahead with what Frank and Mat prefer, thanks!

@msalle:
Ok for you?

**Review by matyasselmeci — APPROVED** [review]
Looks good to me

**Comment by msalle** [comment]
> > Wrong Mischa
> 
> Ooops. Sorry.
> 
> > Hi all,
> > I think we just need to be practical here and propose
> > we go ahead with what Frank and Mat prefer, thanks!
> 
> @msalle:
> Ok for you?

Yes, whether this is the best in the long run I don't know, but we can always change it then. I'd say go for it.

**Review by msalle — APPROVED** [review]
It's fine with me, but given that @ellert  has way more experience with the packaging guidelines, I would prefer to wait for his comments and/or approval.

**Comment by fscheiner** [comment]
> It's fine with me, but given that @ellert has way more experience with the packaging guidelines, I would prefer to wait for his comments and/or approval.

@ellert:
Please give your comment until 2021-06-04.

**Pull request merged**
